README.md

```diff
@@ -49,22 +49,59 @@ Here we list formats that have been used or proposed thus far in the community (
 
 ## Standard Matrices
 
-We benchmark data formats and APIs on standard matrices. These matrices are found at the following S3 bucket.
-
-[TODO add S3 bucket details]
-
-[TODO describe matrices below]
+We benchmark data formats and APIs on standard matrices. These matrices are found in the public S3 bucket `matrix-format-test-data`.
```

Comment:
Why not put a full `s3://` link here?

Reply:
I had the same reaction at first, but also I haven't found a good way to view S3 buckets in a web browser… I keep assuming I am missing something obvious, please lmk if I am 😄 or else, how you meant to link to it?

Reply:
I meant the full `s3://` *path* so one can copy it and browse the directory using aws CLI tool.

Reply:
Yep, I'll update to include the full url.

Reply:
I see; fwiw this works:

```
s3cmd ls matrix-format-test-data
```